frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.tsx

```diff
@@ -358,7 +358,7 @@ class TagEditorParamInner extends Component<Props> {
           <ContainerLabel>
             {t`Default filter widget value`}
             {hasNoDefaultValue && tag.required && (
-              <ErrorSpan>{t`(required)`}</ErrorSpan>
+              <ErrorSpan>({t`required`})</ErrorSpan>
```

Comment:
Just a note that all strings so far have been translated with `()`. This will now change. I agree your proposal makes much more sense. Just drawing attention to the change, and the fact this will have to be updated in all translations.

Example:
```
#: frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.tsx:248
#: frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.tsx:283
#: frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.tsx:325
msgid "(required)"
msgstr "(benötigt)"
```

Reply:
This is actually funny because in one of the previous PRs this was added after an opposite suggestion. So I changed it. And then this change was lost with a rebase :)